sdk/options.go

```diff
@@ -35,6 +36,10 @@ type Options struct {
 	// is closed to signal readiness.
 	Ready chan struct{}
 
+	// Plugins provides a map of pluginName to plugins.Factory to allow plugins
```

Comment:
Nit: instead of:

> Plugins provides a map of pluginName to plugins.Factory to allow plugins to be used by an OPA sdk instance

I'd say:

> Plugins provides a set of plugins.Factory instances that will be registered with the OPA SDK instance.